pahfit/feature_strengths.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from astropy import constants as const
+
+
+def pah_feature_strength(ampl, fwhm, x_0):
+    """
+    Calculate PAH feature strengths.
```

Comment:
Maybe a bit more description here?  Feature strengths could be the central intensity or the integrated area.